wolftpm/tpm2_types.h

```diff
@@ -452,6 +452,8 @@ typedef int64_t  INT64;
     #ifdef WIN32
         #include <windows.h>
         #define XSLEEP_MS(ms) Sleep(ms)
+    #elif defined(WOLFSSL_ESPIDF)
```

Comment:
Would prefer `#elif defined(FREERTOS)`.

Reply:
Agreed using `FREERTOS` here is better. Updated.